lib/dat-worker-pool.rb

```diff
@@ -43,7 +43,7 @@ def start
   end
 
   def shutdown(timeout = nil)
-    @runner.shutdown(timeout)
+    @runner.shutdown(timeout){ caller }
```

Comment:
@kellyredding - Yeah, caller relatively takes a while to build when called. It's probably not worth the efficiency since shutdown is called so little.